extra-multimedia/pipewire/autobuild/defines

```diff
@@ -1,9 +1,8 @@
 PKGNAME=pipewire
 PKGSEC=sound
-PKGDEP="ffmpeg gst-plugins-base-1-0 jack libva v4l-utils"
-BUILDDEP="doxygen graphviz meson ninja xmltoman gtk-doc vim"
 PKGDES="Server and user space API to deal with multimedia pipelines"
+PKGDEP="ffmpeg gst-plugins-base-1-0 jack libva v4l-utils libfdk-aac libldac libopenaptx"
+BUILDDEP="doxygen graphviz meson ninja xmltoman vim"
 
-MESON_AFTER="-Denable-gtk-doc=true \
-             -Dvulkan=false"
+MESON_AFTER="-Dvulkan=false"
```

Comment:
Why drop `gtk-doc`?

Reply:
pipewire's source code doesn't exist any build specs for `gtk-doc`.
if i add it, meson will say `enable-gtk-doc` not exist.